Directory.Build.targets

```diff
@@ -0,0 +1,41 @@
+ <Project ToolsVersion="15.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+ 
+  <!-- Verifies if targets are included into Visual Studio CoreCLR.sln in project which outputs 
+       *.dll or *.exe and sets ShouldCopyVSBuildArtifacts to true  -->
+  <Target Name="_ShouldCopyVSBuildArtifacts" BeforeTargets="CopyVSBuildArtifacts;PostBuildEvent">
+
+    <PropertyGroup>
+      <ShouldCopyVSBuildArtifacts>false</ShouldCopyVSBuildArtifacts>
+    </PropertyGroup>
+
+    <PropertyGroup Condition="($(TargetFileName.EndsWith('.dll')) Or $(TargetFileName.EndsWith('.exe'))) And $(SolutionName) == 'CoreCLR'">
```

Comment:
Having common targets but looking for a particular solution name seems like the wrong direction.

Reply:
I will change it to a more general solution.